.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: "\U0001F41C Bug Report"
+about: Something strange happened? Please tell us about it!
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--- 
+Hi!
+Have you tried searching for your issue on the following forums?
+If you have any questions, please ask them there.
+
+Forums:
+ - TLA⁺ Google Groups forum: https://groups.google.com/g/tlaplus/
+ - Stack Overflow questions (TLA+): https://stackoverflow.com/questions/tagged/tla%2b
```

Comment:
I don't think that the size of the SO community justifies sending people there.

Reply:
Updated